manual/src/cmds/runtime-tracing.etex

```diff
@@ -70,6 +72,33 @@ The full set of events emitted by probes and their documentation can be found in
  section~\ref{Runtime_events}.
 \fi
 
+\subsection{s:runtime-tracing-custom-events}{Custom events}
+The runtime events system supports defining custom events. Identified by a
+globally unique name, they are emitted with payloads of built-in (\texttt{unit},
+\texttt{int}, \texttt{span}) and user-defined types. They are registered using
+the \texttt{Runtime_events.User.register} function. Once an event is registered,
+values for that event are emitted using \texttt{Runtime_events.User.write}.
+
+As consumer and producer can be different programs, they might not have the same
+set of registered events. The event name is used by the consumer to identify
+producer events, with a fallback mechanism for unknown events of built-in types.
+Unknown events for user-defined types are currently ignored, even if the type is
+defined both in the producer and consumer side.
+
+An extensible variant \texttt{tag} provides a way to attach additional data to
+events. The tag is not serialized, so it is only available for \textit{known}
+events.
+
+In summary, there are three cases for the consumer when an event is received:
+\begin{itemize}
+\item event is registered: payload and tag are available.
+\item event is not registered and has a built-in event type (unit, int, span):
+    only the payload is available.
+\item event is not registered and has a custom event type: event is dropped.
+\end{itemize}
+The \texttt{Runtime_events.Type} module provides a way to use built-in event
+types and define new ones using an encoder-decoder pair.
```

Comment:
By "new ones" I understand "new built-in event types". Is it the same as "custom event type", or something different?

Reply:
By "new ones" I meant "new event types" which is indeed the same as "custom event type".

Reply:
```suggestion
types and define custom ones using an encoder-decoder pair.
```